src/DeviceInput/deviceInputSystem.ts

```diff
@@ -480,6 +541,7 @@ export class DeviceInputSystem implements IDisposable {
         this._elementToAttachTo.addEventListener(this._eventPrefix + "move", this._pointerMoveEvent);
         this._elementToAttachTo.addEventListener(this._eventPrefix + "down", this._pointerDownEvent);
         this._elementToAttachTo.addEventListener(this._eventPrefix + "up", this._pointerUpEvent);
+        this._elementToAttachTo.addEventListener("blur", this._pointerBlurEvent);

```

Comment:
I just have to ask that you meant to not add the `this._eventPrefix` to "blur" as you did for the other events, even if I'm sure it's done on purpose!

Reply:
the blur event is not pointer/mouse event, bur an event on the element itself (like `focus`). so it is "standalone" :-)